### ISSUE TITLE
Built-in display does not automatically re-enable when external monitor is unplugged

### ISSUE THREAD

**Issue by hoangnv735** [AlonX2/LightsOut#11]
Thanks for the amazing app. 

I use LightsOut to disable my MacBook's built-in display while working with an external monitor. When I disconnect the HDMI cable for the external monitor, the built-in display remains disabled, leaving me with a black screen and no way to interact with my MacBook.
I must reconnect the external monitor and manually re-enable the built-in display. This is not quite convenient and can lead to undesired situations if one forgets to manually re-enable the built-in display.

It would be more convenient if the app could check and enable the only available display to prevent a blackout situation.

**Comment by EReaso** [comment]
Agreed. When playing with it, I disabled both monitors and needed VNC to re-enable them. Maybe there should be some sort of safety feature that won't disable monitors when there's only one? Or a global shortcut to re-enable all would be great. 

Sorry if I repeated a lot; I didn't read the whole post lol
